### PR TITLE
add no cache header when making FHIR API request

### PR DESCRIPTION
**Pull request by achen2401**
part of https://www.pivotaltracker.com/story/show/180042213

### CODE REVIEW

**Review by pbugni — APPROVED** [review]

File: src/utils/executeELM.js [hunk 1]
```diff
@@ -103,7 +103,14 @@ function doSearch(client, release, type, collector) {
   const resources = [];
   const uri = `${type}?${params}`;
   return new Promise((resolve) => {
-    const results = client.patient.request(uri, {
+    const results = client.patient.request({
+        url: uri,
+        headers: {
+          'Cache-Control': 'no-cache, no-store, max-age=0',
+          'Expires': 'Tue, 01 Jan 1980 1:00:00 GMT',
+          'Pragma': 'no-cache'
```
Comment (ivan-c) [inline src/utils/executeELM.js:111]: Did you find more than `Cache-Control': 'no-cache'` was needed? Are these other headers necessary, and what do they do?

**Review by ivan-c — APPROVED** [review]
Had a question, but no immediate blockers

File: src/utils/executeELM.js (hunk 1 above)
Comment (achen2401) [reply src/utils/executeELM.js:111]: I guess my concern is that not all browsers treat the HTTP directives the same:  I see a discussion here:  https://stackoverflow.com/questions/49547/how-do-we-control-web-page-caching-across-all-browsers

Comment (ivan-c) [reply src/utils/executeELM.js]: If I understand what's going on correctly, HAPI is caching results, so this shouldn't be affected by browsers

Comment (ivan-c) [reply src/utils/executeELM.js]: Thanks for the change!

**Pull request merged**